packages/backend/server/src/data/migrations/utils/prompts.ts

```diff
@@ -492,6 +492,12 @@ content: {{content}}`,
       },
     ],
   },
+  {
+    name: 'Create a presentation:workflow',
+    action: 'Create a presentation:workflow',
+    model: 'Create a presentation',
```

Comment:
model what?

Reply:
used for workflow, in workflow api we will use model name to find workflow graph

Reply:
i. added a prefix `workflow:` to make it clearly